Gemfile

```diff
@@ -8,6 +8,8 @@ gem "chef-config", path: "chef-config" if File.exist?(File.expand_path("../chef-
 # Ensure that we can always install rake, regardless of gem groups
 gem "rake"
 
+gem "ffi", github: "lamont-granquist/ffi", branch: "lcg/rb_gc_guard_ptr" if RUBY_PLATFORM.downcase =~ /aix/
```

Comment:
Can you add a comment with a link to https://github.com/ffi/ffi/pull/490 so we'll know when we can go back to the mainline? Also, I'd be fine just running everything from your branch instead of branching for AIX.